src/Umbraco.Core/Services/Implement/ContentService.cs

```diff
@@ -757,6 +757,11 @@ public OperationResult Save(IContent content, int userId = Constants.Security.Su
             if (publishedState != PublishedState.Published && publishedState != PublishedState.Unpublished)
                 throw new InvalidOperationException("Cannot save (un)publishing content, use the dedicated SavePublished method.");
 
+            if (content.Name.Length > 255)
```

Comment:
Looks like these changes are making some tests fail. Since we don't have much validation for models at the service layer, this is sort of a 'first'. Because of that, there are unit tests that don't actually specify a `Name` at all on the content which means this is causing a null reference exception. 

So all of these checks in these 3 places need to instead be:

`if (content.Name != null && content.Name.Length > 255)`